db_upgrade_scripts/mosip_ida/sql/1.2.1.0_to_1.3.0_rollback.sql

```diff
@@ -1,28 +1,38 @@
 -- ------------------------------------------------------------------------------------------
--- Rollback script for reverting Spring Batch version 5.0 migration as part of Java 21 migration.
+-- Rollback script for Reverting Spring Batch version upgrade changes.
 -- ------------------------------------------------------------------------------------------
 
--- Revert the changes to BATCH_STEP_EXECUTION
-ALTER TABLE BATCH_STEP_EXECUTION ALTER COLUMN START_TIME SET NOT NULL;
+-- Reverse the addition of the CREATE_TIME column
 ALTER TABLE BATCH_STEP_EXECUTION DROP COLUMN CREATE_TIME;
 
--- Revert the changes to BATCH_JOB_EXECUTION_PARAMS
+-- Reverse the modification of START_TIME column
+ALTER TABLE BATCH_STEP_EXECUTION ALTER COLUMN START_TIME SET NOT NULL;
+
+-- Reverse the removal of DATE_VAL, LONG_VAL, and DOUBLE_VAL columns
 ALTER TABLE BATCH_JOB_EXECUTION_PARAMS ADD COLUMN DATE_VAL TIMESTAMP;
 ALTER TABLE BATCH_JOB_EXECUTION_PARAMS ADD COLUMN LONG_VAL BIGINT;
 ALTER TABLE BATCH_JOB_EXECUTION_PARAMS ADD COLUMN DOUBLE_VAL DOUBLE PRECISION;
-ALTER TABLE BATCH_JOB_EXECUTION_PARAMS ALTER COLUMN PARAMETER_TYPE TYPE CHAR(6);
+
 ALTER TABLE BATCH_JOB_EXECUTION_PARAMS RENAME PARAMETER_TYPE TO TYPE_CD;
+
+-- Reverse the modifications and renaming of KEY_NAME to PARAMETER_NAME
 ALTER TABLE BATCH_JOB_EXECUTION_PARAMS ALTER COLUMN PARAMETER_NAME TYPE VARCHAR(100);
 ALTER TABLE BATCH_JOB_EXECUTION_PARAMS RENAME PARAMETER_NAME TO KEY_NAME;
+
+-- Reverse the modifications and renaming of STRING_VAL to PARAMETER_VALUE
 ALTER TABLE BATCH_JOB_EXECUTION_PARAMS ALTER COLUMN PARAMETER_VALUE TYPE VARCHAR(250);
 ALTER TABLE BATCH_JOB_EXECUTION_PARAMS RENAME PARAMETER_VALUE TO STRING_VAL;
+
+-- Reverse the dropping of JOB_CONFIGURATION_LOCATION column
 ALTER TABLE BATCH_JOB_EXECUTION ADD COLUMN JOB_CONFIGURATION_LOCATION VARCHAR(2500);
 
--- Drop the created sequences
-DROP SEQUENCE BATCH_STEP_EXECUTION_SEQ;
-DROP SEQUENCE BATCH_JOB_EXECUTION_SEQ;
-DROP SEQUENCE BATCH_JOB_SEQ;
+-- Reverse the creation of indexes
+DROP INDEX IF EXISTS idx_job_name;
+DROP INDEX IF EXISTS idx_job_key;
 
--- Drop the created indexes
-DROP INDEX idx_job_name;
-DROP INDEX idx_job_key;
+TRUNCATE TABLE batch_job_execution;
+TRUNCATE TABLE batch_job_execution_context;
+TRUNCATE TABLE batch_job_execution_params;
+TRUNCATE TABLE batch_job_instance;
+TRUNCATE TABLE batch_step_execution;
+TRUNCATE TABLE batch_step_execution_context;
```

Comment:
Check whether there is any impact of truncating the data.

Reply:
it will clear only batch job data